modules/ztp-sno-siteconfig-config-reference.adoc

```diff
@@ -48,6 +48,9 @@ For example, `policygentemplates/common-ranGen.yaml` applies to all clusters wit
 For three-node deployments, define three hosts.
 For standard deployments, define three hosts with `role: master` and two or more hosts defined with `role: worker`.
 
+|`spec.clusters.nodes.nodeLabels`
+|Specify custom roles to your nodes in your managed clusters. These are additional roles which are not used by any {product-title} components, only by the user. When you add a custom role, it can be associated with a custom machine config pool that references a specific configuration for that role. Adding your custom labels or roles during installation makes the deployment process more effective and prevents the need for additional reboots after the installation is complete.
```

Comment:
```suggestion
|Specify custom roles for nodes in your managed clusters. These are additional roles that are not used by any {product-title} components, only by the user. When you add a custom role, it can be associated with a custom machine config pool that references a specific configuration for that role. Adding custom labels or roles during installation makes the deployment process more effective and prevents the need for additional reboots after the installation is complete.
```